client/keys/parse.go

```diff
@@ -17,14 +17,7 @@ import (
 )
 
 func bech32Prefixes(config *sdk.Config) []string {
```

Comment:
why not just eliminate completely?

Reply:
Didn't want to impact too much, but yeah will do